pebblo/app/service/service.py

```diff
@@ -53,7 +53,7 @@ def _write_pdf_report(self, final_report):
         full_file_path = get_full_path(current_app_report_file_path)
         report_obj.generate_report(
             data=final_report,
-            outputPath=full_file_path,
+            output_path=full_file_path,
             format=report_format,
```

Comment:
Correct the argument name(format -> format_string)

Reply:
Resolved